helm/amazon-ec2-metadata-mock/templates/tests/test-aemm-service.yaml

```diff
@@ -48,10 +48,16 @@ spec:
     - "bash"
     - "-c"
     - |
+      set -x
```

Comment:
do we really need to `set -x` before every var or can we just do it once at the start?

Reply:
I didn't mean to check this in.. I put it to debug an issue :/ 

Will remove in next revision